pom.xml

```diff
@@ -41,136 +44,37 @@
-                                <id>sign-artifacts</id>
-                                <phase>verify</phase>
-                                <goals>
-                                    <goal>sign</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                    <plugin>
-                        <artifactId>maven-source-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <goals>
-                                    <goal>jar-no-fork</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                    <plugin>
-                        <artifactId>maven-javadoc-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <goals>
-                                    <goal>jar</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-
-    </profiles>
-
-    <dependencies>
-        <dependency>
-            <groupId>com.basis.lib</groupId>
-            <artifactId>BBj.jar</artifactId>
-            <scope>system</scope>
-            <version>1.0</version>
-            <systemPath>${jar-directory}/BBj.jar</systemPath>
-        </dependency>
-        <dependency>
-            <groupId>com.basis.lib</groupId>
-            <artifactId>BBjStartup.jar</artifactId>
-            <scope>system</scope>
-            <version>1.0</version>
-            <systemPath>${jar-directory}/BBjStartup.jar</systemPath>
-        </dependency>
-        <dependency>
-            <groupId>com.basis.lib</groupId>
-            <artifactId>BBjUtil.jar</artifactId>
-            <scope>system</scope>
-            <version>1.0</version>
-            <systemPath>${jar-directory}/BBjUtil.jar</systemPath>
-        </dependency>
-
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-engine</artifactId>
-            <version>5.3.2</version>
-            <scope>test</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>com.google.code.gson</groupId>
-            <artifactId>gson</artifactId>
-            <version>2.10.1</version>
-        </dependency>        
-    </dependencies>
+    <modules>
+        <module>dwcj-engine</module>
+        <module>dwcj-widgets/dwcj-markdown-widget</module>
+        <module>dwcj-widgets/dwcj-code-widget</module>
+        <module>dwcj-widgets/dwcj-terminal-widget</module>
+    </modules>
+
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.commonmark</groupId>
```

Comment:
@StephanWald Why do we need the `commonmark` in the parent POM

Reply:
agreed. Moved to the markdown widget.

How about gson - leave that one in the parent for it being used more often?

Reply:
Let's leave it to the parent. I can see us utilizing that one a lot in the future.